neo/Prompt/Commands/Wallet.py

```diff
@@ -108,11 +107,10 @@ def execute(self, arguments):
 
             if PromptData.Wallet:
                 PromptData.Prompt.start_wallet_loop()
-                return
+                return PromptData.Wallet
 
         else:
             print("Please specify a path")
-            return
```

Comment:
I saw this, but I'm mitigated about whether it should stay as is or not. 
If someone calls `CommandWalletCreate.execute()` directly it would skip the argument validation.

@ixje Any advice?

Reply:
Similar to my previous realization, I understand now that this should be caught by your new logic concerning required input. I think what you have should be good but then we should remove redundant code.